lib/radiator_web/helpers/auth_helpers.ex

```diff
@@ -0,0 +1,55 @@
+defmodule RadiatorWeb.Helpers.AuthHelpers do
+  @moduledoc """
+  Authentication helper functions for the web layer.
+  """
+  alias Radiator.Auth
+  alias Radiator.Auth.User
+
+  def session_response(user = %User{}, token) do
+    %{
+      username: user.name,
+      token: token,
+      expires_at: Auth.Guardian.get_expiry_datetime(token)
+    }
+  end
+
+  def current_user(conn) do
+    conn.assigns[:current_user]
```

Comment:
okay, so you finally did go for current_user? I would prefer we unify to `authenticated_user` or `current_user` back in the login controllers.